examples/components/menu/basic_usage.ts

```diff
@@ -0,0 +1,18 @@
+import {Component} from "angular2/core";
+import {MATERIAL_DIRECTIVES} from "ng2-material/all";
+
+@Component({
+  selector: 'menu-basic-usage',
+  templateUrl: 'examples/components/menu/basic_usage.html',
+  styleUrls: ['examples/components/menu/basic_usage.css'],
+  directives: [MATERIAL_DIRECTIVES]
+})
+export default class MenuBasicUsage {
+ display: boolean = false;
+
+test()
```

Comment:
wrong indent. What is your text editor ? It should have some option to auto-align on save, a pretty useful tool if it's well configured